snuba/settings/__init__.py

```diff
@@ -175,7 +175,7 @@
 SUBSCRIPTIONS_ENTITY_BUFFER_SIZE: Mapping[str, int] = {}  # (entity name, buffer size)
 
 # Temporary setting for subscription scheduler test
-SUBSCRIPTIONS_SCHEDULER_LOAD_FACTOR = 2
+SUBSCRIPTIONS_SCHEDULER_LOAD_FACTOR = 3
```

Comment:
what about 4 or 5 ?

Reply:
Yeah we can probably handle 3 easily, just curious in see how this goes step by step. We will probably not be on 3 for long.